packages/realm-web/src/index.ts

```diff
@@ -38,26 +36,11 @@ export function app(id: string) {
     }
 }
 
-/**
- * Handle an OAuth 2.0 redirect.
- *
- * @param location An optional location to use (defaults to the windows current location).
- * @param storage Optional storage used to save any results from the location.
- */
-export function handleAuthRedirect(
-    location = window.location,
-    storage = createDefaultStorage(),
-) {
-    try {
-        const queryString = location.hash.substr(1); // Strip the initial # from the hash
-        OAuth2Helper.handleRedirect(queryString, storage);
-    } catch (err) {
-        // Ensure calling this never throws: It should not interrupt a users flow.
-        console.warn(err);
-    }
-}
-
 export { App };
 export { Credentials } from "./Credentials";
 export { User, UserState } from "./User";
-export { createDefaultStorage } from "./storage";
+export * from "./environment";
+
+// Export a selection of the bson types
+import { ObjectId } from "bson";
+export const BSON = { ObjectId };
```

Comment:
Why not export the full `BSON`? It seems confusing to introduce 2 different BSONs

Reply:
Technically it won't be 2 different BSONs, when importing from "bson" it will return the same module object as any other API which will be importing "bson".

I choose to go with the single type, because that was the only type the MongoDB service expected, but now that this has changed we might actually choose to leave this re-export out entirely?

Reply:
I figured out that this export turns out to be the cause of a ///-refrence to node in out index.d.ts, so I'll remove this for now and add it if someone requests it.